src/model/OWL.pl

```diff
@@ -237,30 +237,45 @@
   triple(R,owl:onProperty,P).
 
 is_restriction1(R,value(P,O)) ?+>
-  triple(R,owl:onProperty,P),
-  triple(R,owl:hasValue,O).
+  triple(R,owl:hasValue,O),
+  triple(R,owl:onProperty,P).
 
 is_restriction1(R,min(P,M,O)) ?+>
+  triple(R,owl:minQualifiedCardinality,M),
   triple(R,owl:onProperty,P),
```

Comment:
same here, I put P first because in most cases P will be instantiated when this is called.
Why the reordering?